src/js/_enqueues/wp/password-strength-meter.js

```diff
@@ -22,16 +22,16 @@ window.wp = window.wp || {};
 		 *
 		 * @since 3.7.0
 		 *
-		 * @param {string} password1 The subject password.
-		 * @param {Array}  blacklist An array of words that will lower the entropy of
-		 *                           the password.
-		 * @param {string} password2 The password confirmation.
+		 * @param {string} password1       The subject password.
+		 * @param {Array}  disallowed_list An array of words that will lower the entropy of
```

Comment:
```suggestion
		 * @param {Array}  disallowedList An array of words that will lower the entropy of
```

In JS we use camelCase for variable/method names. Any reason for not using `blocklist` to match `userInputBlocklist()`?

Reply:
I originally had `blocklist` and changed it. I struggle with "block list" vs. "blocklist" in that I prefer the former.

`user_inputs` could work though. Do you suggest that for consistency with the library, or `userInputs` for coding standards consistency?

Reply:
Went with `disallowedList`.